tokio/src/loom/std/atomic_usize.rs

```diff
@@ -16,6 +16,12 @@ impl AtomicUsize {
         AtomicUsize { inner }
     }
 
+    #[cfg(all(feature = "parking_lot", not(all(loom, test)),))]
+    pub(crate) const fn const_new(val: usize) -> AtomicUsize {
+        let inner = UnsafeCell::new(std::sync::atomic::AtomicUsize::new(val));
+        AtomicUsize { inner }
+    }
```

Comment:
Can we not just add the `const` marker to the ordinary `new` function in this case? It is an internal type, so it introduces no backwards compatibility challenges, and the functions are identical.